src/mame/ussr/juku.cpp

```diff
@@ -102,7 +110,11 @@ class juku_state : public driver_device
 
 private:
 	required_device<i8080a_cpu_device> m_maincpu;
-	required_device<address_map_bank_device> m_bank;
+	required_region_ptr<uint8_t> m_rom;
+	optional_region_ptr<uint8_t> m_exp;
+	required_shared_ptr<uint8_t> m_ram;
+	optional_shared_ptr_array<uint8_t, 1> m_ext;
```

Comment:
You could change this into a `memory_share_array_creator<uint8_t, 1>` and supply the size at construction to simplify things.